nep-15.mediawiki

```diff
@@ -49,7 +49,7 @@ The <code>groups</code> field is an array of <code>Group</code> objects which re
 
 The <code>features</code> field will be used for future expansion and should always be an empty object for now.
 
-The <code>supportedstandards</code> field describes the [[obsolete/nep-10.mediawiki|NEP-10]] values.
+The <code>supportedstandards</code> field describes which NEP standard it supports. It must be an array. The format should be all in upper cases which has "-" between <code>NEP</code> and standard number. For example: <code>"supportedstandards": ["NEP-11", "NEP-17"]</code>.
```

Comment:
And the standard here can be not only NEP, but also others, such as RFC.

Reply:
> And the standard here can be not only NEP, but also others, such as RFC.

Done.